### PR TITLE
Update docs 0912 for action tutorials

### PR DESCRIPTION
**Pull request by csantanapr**
Addressed multiple issues
- update sample to reflect correct value for 20sec
- Fix python sample
- rename sequence action name
- workaround for sequence not working with CLI #1248 
- workaround for user to see correct Dockerfile to use, will remove once sdk updated

### CODE REVIEW

**Comment by rabbah** [comment]
> update sample to reflect correct value for 20sec
How about changing the text to two seconds instead - for an example waiting 20 seconds is not necessary or ideal.

**Comment by rabbah** [comment]
> workaround for user to see correct Dockerfile to use, will remove once sdk updated

sdk is updated - do you mean promoted on Bluemix?

**Comment by rabbah** [comment]
> + $ chmod +x buildAndPush.sh

This is also a workaround against a known go cli bug - will link it so that we remove this when the issue is fixed.

File: docs/actions.md [hunk 1]
```diff
@@ -549,6 +553,31 @@ For the instructions that follow, assume that the Docker user ID is `janesmith`
...
+          gcc \
+          libc-dev \
+  ### Compile source file(s)
+  && cd /action; gcc -o exec example.c \
+  && apk del .build-deps
+
+  CMD ["/bin/bash", "-c", "cd actionProxy && python -u actionproxy.py"]  
+  ```
+  ```
+  $ chmod +x buildAndPush.sh
```
Comment (rabbah) [inline docs/actions.md]: issue https://github.com/openwhisk/openwhisk/issues/1129.

**Comment by csantanapr** [comment]
@rabbah review the last two commits.
Now that docker sdk it's promoted workaround no longer need it.

**Comment by rabbah** [comment]
Merged as https://github.com/openwhisk/openwhisk/commit/2045795e1c4f2d9c1e7237a9d8716ec921945428.

**Pull request closed without merging**